package_control/package_manager.py

```diff
@@ -1292,8 +1292,9 @@ def install_package(self, package_name, is_dependency=False):
 
             # Here we don't use .extractall() since it was having issues on OS X
             overwrite_failed = False
-            extracted_paths = []
-            for path in package_zip.namelist():
+            extracted_paths = set()
+            for info in package_zip.infolist():
```

Comment:
Is there a performance difference in the call to `infolist()` versus `namelist()`, or is it just in the usage of `set()`?

If it is important to call `infolist()`, it would be good to add a comment explaining why we use it so it isn't refactored away in the future to be more "pythonic".

Reply:
It simply avoids creating a whole new list from `infolist()`. It's implemented as `[info.filename for info in infolist()]`. So it's not very important, but I felt that if the zip `info` is ever going to be required in this loop it'll be done right if it's already available - rather than someone querying the info by filename for every file.